Formula/forgit.rb

```diff
@@ -0,0 +1,28 @@
+class Forgit < Formula
+  desc "Interactive git commands in the terminal"
+  homepage "https://github.com/wfxr/forgit"
+  url "https://github.com/wfxr/forgit/releases/download/23.04.0/forgit-23.04.0.tar.gz"
+  sha256 "89826d12b551829c4bce396551b1505c0604ba29130f4b8e2f842c2ca0b4c9e3"
+  license "MIT"
+
+  depends_on "fzf"
+
+  def install
+    bin.install "bin/git-forgit"
+    bash_completion.install "completions/git-forgit.bash"
+    prefix.install "forgit.plugin.zsh"
```

Comment:
Let's use `pkgshare` since that's where other formulae install Zsh plugins.
```suggestion
    inreplace "forgit.plugin.zsh", 'FORGIT="$INSTALL_DIR', "FORGIT=\"#{opt_prefix}"
    pkgshare.install "forgit.plugin.zsh"
    pkgshare.install_symlink "forgit.plugin.zsh" => "forgit.plugin.sh"
```

Reply:
@carlocab Great, thanks a lot for the feedback. I updated the branch with your sugesstions.